src/envoy/http/authn/authenticator_base.cc

```diff
@@ -29,6 +29,24 @@ namespace Http {
 namespace Istio {
 namespace AuthN {
 
+namespace {
+// The default header name for an exchanged token
+static const std::string kExchangedTokenHeaderName = "ingress-authorization";
+
+// Return whether the header for an exchanged token is found
+bool FindHeaderOfExchangedToken(const iaapi::Jwt& jwt) {
```

Comment:
IsExchangedToken or IsAPToken may be a better name.

Reply:
FindHeaderOfExchangedToken() accurately describes what this function does.